src/main/scala/nl.knaw.dans.easy.stage/dataset/DepositorInfo.scala

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (C) 2015 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.stage.dataset
+
+import java.io.FileNotFoundException
+import java.nio.charset.StandardCharsets
+import java.nio.file.{ Files, NoSuchFileException, Path }
+
+import nl.knaw.dans.lib.error._
+import nl.knaw.dans.lib.logging.DebugEnhancedLogging
+import nl.knaw.dans.lib.string._
+import org.apache.commons.lang.BooleanUtils
+
+import scala.util.Try
+import scala.xml.{ Elem, XML }
+
+case class DepositorInfo(acceptedLicense: Boolean, privacySensitiveRemark: String, messageFromDepositor: Option[String])
+
+object DepositorInfo extends DebugEnhancedLogging {
+  def apply(depositorInfoDir: Path): DepositorInfo = {
+    lazy val depositDir = depositorInfoDir.getParent.getParent.getParent.toFile.getName
```

Comment:
Is this the depositId? Isn't that available as a parameter from `EasyStageDataset`?